### DISCUSSION TITLE
rec训练配置文件 svtr配置是 warmup_epoch: 5，resnet没这个配置，把这个配置改为0，有什么不好的影响吗？能加快训练？

### DISCUSSION THREAD

**Discussion by nissansz** [PaddlePaddle/PaddleOCR#13077] [Q&A]
rec训练配置文件 svtr配置是 warmup_epoch: 5，resnet没这个配置，把这个配置改为0，有什么不好的影响吗？能加快训练？

**Comment by SWHL** [comment]
hello， 建议不要在PaddleOCR项目中疯狂提问了哈。遇到问题自己多想想，多半就能解决。

**Comment by nissansz** [comment]
如果自己能解决，还来提问干什么呢？
能不能给一点到位的答复，而不是一味的将bug转为dicussion, 感觉一点用处没有。

**Reply by SWHL** [reply]
开源项目都是大家用爱发电在维护，我们没有责任和义务帮你解决这种问题

仓库README中已经声明，issue只用来放置仓库bug，你的问题都不是bug。
后面如果从discusssion中发现是bug，会转为issue的。

同时，建议看一下如何有效提问 → [link](https://github.com/ryanhanwu/How-To-Ask-Questions-The-Smart-Way/blob/main/README-zh_CN.md)

**Comment by nissansz** [comment]
这个svtr根本就是没训练两下就自动报错了

**Reply by SWHL** [reply]
如果报错了，那你提问的是什么呢？想让别人帮助你，起码要先把运行环境，报错信息，自己做了哪些查找之后，最后不得已才来提问吧！

 你这只说自己遇到了错误，谁来帮我解决一下呀！

**Comment by nissansz** [comment]
转到discussion，貌似根本一个人都不会看。根本就没人回复过

**Reply by SWHL** [reply]
提到issue，就会有人回复了？你看看Discussions中，一直都有小伙伴提问和回复的啊
有么有可能是自己提的问题，没人回复呢

**Comment by nissansz** [comment]
我是发现只要你转到discussion，根本就没人看。

**Comment by SWHL** [comment]
建建议再仔细发现发现哈

**Comment by nissansz** [comment]
还不如把discussion, bug全部禁用算了。

**Reply by SWHL** [reply]
就因为没人回答你的问题？你有提问的权利，但请谨慎使用
